core/designsystem/src/main/java/com/acon/core/designsystem/component/textfield/AconTextField.kt

```diff
@@ -0,0 +1,173 @@
+import androidx.compose.foundation.gestures.detectTapGestures
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusManager
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import androidx.compose.ui.focus.onFocusChanged
+import androidx.compose.ui.input.pointer.pointerInput
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.acon.core.designsystem.theme.AconColors
+import com.acon.core.designsystem.theme.AconTheme
+
+@Composable
+fun AconTextField(
+    status: TextFieldStatus,
+    modifier: Modifier = Modifier,
+    text: String = "text",
+    onTextChanged: (String) -> Unit = {},
+    placeholder: String = "",
+    onFocusChanged: (Boolean) -> Unit = {},
+    keyboardOptions: KeyboardOptions = KeyboardOptions.Default,
+    keyboardActions: KeyboardActions = KeyboardActions.Default,
+) {
+
+    val focusRequester = remember { FocusRequester() }
+    val focusManager = LocalFocusManager.current
+
+    val backgroundColor = when (status) {
+        TextFieldStatus.Inactive -> AconColors.Gray8
+        TextFieldStatus.Focused -> AconColors.Gray9
+        TextFieldStatus.Active -> AconColors.Gray9
+        TextFieldStatus.Error -> AconColors.Gray9
+        TextFieldStatus.Disabled -> AconColors.Gray7
+    }
+
+    val borderColor = when (status) {
+        TextFieldStatus.Inactive -> AconColors.Gray6
+        TextFieldStatus.Focused -> AconColors.Gray6
+        TextFieldStatus.Active -> AconColors.Error_red1
+        TextFieldStatus.Error -> AconColors.Gray6
+        TextFieldStatus.Disabled -> AconColors.Gray6
+    }
+
+    val textColor = when (status) {
+        TextFieldStatus.Inactive -> AconColors.Gray5
+        TextFieldStatus.Focused -> AconColors.Gray5
+        TextFieldStatus.Active -> AconColors.White
+        TextFieldStatus.Error -> AconColors.Gray6
+        TextFieldStatus.Disabled -> AconColors.Gray5
+    }
+
+    val isEnabled = status != TextFieldStatus.Disabled
```

Comment:
```suggestion
    val isEnabled = remember(status) { derivedStateOf {
        status != TextFieldStatus.Disabled
    }}
```

요렇게 바꿔줘서 잦은 상태 업데이트를 줄여줄 수 있을 것 같아요!!